unit_test/live_model_test.cc

```diff
@@ -536,27 +534,11 @@ BOOST_AUTO_TEST_CASE(live_model_logger_receive_data) {
     }
 
     Verify(Method((*mock_observation_sender), init)).Exactly(1);
-    Verify(Method((*mock_interaction_sender), init)).Exactly(1);
-    Verify(Method((*mock_decision_sender), init)).Exactly(2);
+    Verify(Method((*mock_interaction_sender), init)).Exactly(3);
   }
-  //std::string recorded_interactions_all;
-  //for (size_t i = 0; i < recorded_interactions.size(); ++i) {
-  //  recorded_interactions_all += recorded_interactions[i];
-  //  if (i + 1 < recorded_interactions.size()) {
-  //    recorded_interactions_all += '\n';
-  //  }
-  //}
-
-  //std::string recorded_observations_all;
-  //for (size_t i = 0; i < recorded_observations.size(); ++i) {
-  //  recorded_observations_all += recorded_observations[i];
-  //  if (i + 1 < recorded_observations.size()) {
-  //    recorded_observations_all += '\n';
-  //  }
-  //}
-
-  //BOOST_CHECK_EQUAL(recorded_interactions_all, expected_interactions);
-  //BOOST_CHECK_EQUAL(recorded_observations_all, expected_observations);
+  // TODO deserialize and check contents with expected
```

Comment:
from what I understand the interactions and observations are small and get batched into one message, hence checking if `recorded_interactions.size()` and `recorded_observations.size()` `>= 1`